backport-changelog/6.8/7498.md

```diff
@@ -0,0 +1,4 @@
+https://github.com/WordPress/wordpress-develop/pull/7498
+
+* https://github.com/WordPress/gutenberg/pull/60622
+* https://github.com/WordPress/gutenberg/pull/65181
```

Comment:
Why is this added twice?

Reply:
One is for the REST API, and the other is for inline comments.